src/Users/Collection.php

```diff
@@ -2,12 +2,10 @@
 
 namespace ATDev\RocketChat\Users;
 
-use Doctrine\Common\Collections\ArrayCollection;
-
 /**
  * User collection class
  */
-class Collection extends ArrayCollection
+class Collection extends \ATDev\RocketChat\Common\Collection
```

Comment:
The same I was talking about above. Do we really need message-related properties here?

Reply:
No, we don't need it. I removed message-related property from Common\Collection.